plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2319,8 +2319,8 @@ workspace.textures.delete_selected=Delete selected
 workspace.textures.export_selected=Export selected ...
 workspace.textures.confirm_deletion_message=<html>Are you sure that you want to delete this file?<br>NOTE: If you use this file anywhere, you might have broken textures!
 workspace.textures.select_dupplicate_type=Select the type of texture you would like to duplicate into:
-workspace.textures.category_blocks=Textures for blocks:
-workspace.textures.category_items=Textures for items:
+workspace.textures.category_block=Textures for blocks:
```

Comment:
Change this to workspace.textures.category.block and others too, as other autogenerated keys split with dot too, to streamline this